src/_inc/footer.htm

```diff
@@ -14,38 +14,18 @@
     <!-- JAVASCRIPTS -->
     <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDCn8TFXGg17HAUcNpkwtxxyT9Io9B_NcM" defer></script>
     <script src="assets/plugins/jquery/jquery.min.js"></script>
+    <script src="assets/plugins/bootstrap/js/bootstrap.bundle.js"></script>
```

Comment:
Please use bootstrap.bundle.min.js

Reply:
Ok, thanks for reviewing, I am doing it now.